blockchain/types/tx_internal_data.go

```diff
@@ -74,6 +74,11 @@ const (
 	TxValueKeyCodeFormat
 )
 
+const (
```

Comment:
If you want to declare this, how about creating a new type like `TxTypeMask` or `TxTypeFilter`?

Reply:
I defined a TxTypeMask and apply it. Thanks @jeongkyun-oh .